string action
int32 position
int32 speed
int32 force
bool useContactDetection
